aict_tools/scripts/train_disp_regressor.py

```diff
@@ -138,7 +138,9 @@ def main(configuration_path, signal_path, predictions_path, disp_model_path, sig
             cv_disp_prediction = np.exp(cv_disp_prediction)
 
         sign_classifier.fit(cv_x_train, cv_sign_train)
-        cv_sign_prediction = sign_classifier.predict(cv_x_test)
+        cv_sign_probabilities = sign_classifier.predict_proba(cv_x_test)[:, 1]
```

Comment:
This is unused and evaluates the model twice (which is an expensive operation). You only need the score.

Reply:
I didn't know `sklearn.metrics.roc_curve()` worked with `y_score` values < 0.
Fixed it.